File: 465a/main.go
package main

import (
	"bufio"
	. "fmt"
	"io"
	"os"
)

func run(r io.Reader, w io.Writer) {
	in := bufio.NewReader(r)
	out := bufio.NewWriter(w)
	defer out.Flush()
	var a int
	Fscan(in, &a)
	var s string
	Fscan(in, &s)
	ans := 0
	for _, v := range s {
		if v == '0' {
			Fprintln(out, ans+1)
			return
		}
		ans += 1
	}
	Fprintln(out, ans)
	return

}

func main() {
	run(os.Stdin, os.Stdout)
}

File: 465a/main.cpp
#include <iostream>
using namespace std;

int main() {
    int a;
    cin >> a;
    char L[a];
    int ans = 0;

    for (int i = 0; i < a; i ++){
        cin >> L[i];
        if (L[i] == '0') {
            cout << ans + 1 << '\n';
            return 0;
        };
        ans ++;

    };
    cout << ans << '\n';
    return 0;

}
